### DISCUSSION TITLE
Capturing break-time

### DISCUSSION THREAD

**Discussion by beriknl** [kimai/kimai#2642] [On-Premise: Installation & Updates]
I noticed the timesheet calculates time between two date fields. Is it possible to capture breaktime? 
https://www.kimai.org/store/custom-fields-bundle.html
I noticed the custom fields plugin. Would this allow for it?

**Comment by kevinpapst** [comment]
Support for a break time will be added in the future (already on the roadmap).

You could add a custom duration textfield for the break time, but that field would only be shown in export and lists, but you couldn't really use it for reporting inside Kimai.

**Reply by cicibai** [reply]
When can we expect the next upgrade? I would really appreciate a rough time frame :)

**Reply by kevinpapst** [reply]
I am not giving any timeframe for unpaid features. Sorry, but that only leads to wrong expectations.

**Reply by cicibai** [reply]
Not even a super rough estimate? One month? One year? I promise I won't hold it against you if it turns out to be off - I'm just trying to decide if I should start working on this feature on my own or wait for the update. Please let me know!

**Reply by kevinpapst** [reply]
If you want a motivation: I'd love to work on that feature together with you, so if you start with a PR => I am here to help with input!

**Reply by cicibai** [reply]
Let's do it! I'll go ahead with that right now.

**Reply by Mytronus** [reply]
Hi together,
is there any progress on this?
